test/commands/clear-account.test.js

```diff
@@ -0,0 +1,39 @@
+const { expect, test } = require("@oclif/test");
+const sinon = require("sinon");
```

Comment:
we already have jest, so you should be use jest to create the stubs, is there any reason for bringing in sinon as well?

Reply:
see https://jestjs.io/docs/en/mock-functions